less/css/woocommerce.less

```diff
@@ -418,5 +418,14 @@
 			float: left;
 		}
 	}
+	
+	&.post-type-archive-product
```

Comment:
Did you mean a comma or a bracket? I'm going to presume a bracket. If that's not right, could you please elaborate?